### DISCUSSION TITLE
Automatic Cutting trimesh

### DISCUSSION THREAD

**Discussion by ziwenjie** [cnr-isti-vclab/PyMeshLab#211] [Q&A]
Hi:
Meshlab has the function of cutting trimesh automatically in batchmode?
Thank you!
yours sincerely

**Comment by alemuntoni** [comment]
Hi, what do you mean with "cutting trimesh"?
You can for example select vertices and faces conditionally ([`compute_selection_by_condition_per_face`](https://pymeshlab.readthedocs.io/en/latest/filter_list.html#compute_selection_by_condition_per_face) and [`compute_selection_by_condition_per_vertex`](https://pymeshlab.readthedocs.io/en/latest/filter_list.html#compute_selection_by_condition_per_vertex)) and then remove only the selected ones ([`meshing_remove_selected_vertices_and_faces`](https://pymeshlab.readthedocs.io/en/latest/filter_list.html#meshing_remove_selected_vertices_and_faces)).
